cmake-tool/helpers/application_settings.cmake

```
@@ -35,9 +36,9 @@ function(ApplyData61ElfLoaderSettings kernel_platform kernel_sel4_arch)
         set(ElfloaderMode "hypervisor" CACHE STRING "" FORCE)
         set(ElfloaderMonitorHook ON CACHE BOOL "" FORCE)
     endif()
-    if((KernelPlatformImx8mm-evk OR KernelPlatformImx8mq-evk) AND KernelSel4ArchAarch32)
+    if((KernelPlatformImx8mm-evk OR KernelPlatImx8mq) AND KernelSel4ArchAarch32)
         set(ElfloaderArmV8LeaveAarch64 ON CACHE BOOL "" FORCE)
-        # This applies to imx8mm and imx8mq when in aarch32 configuration
+        # This applies to imx8m, imx8mq and maaxboard when in aarch32 configuration
```

```suggestion
        # This applies to imx8m, imx8mq (EVK and MaaxBoard) when in aarch32 configuration
```

I think we can leave `KernelPlatImx8mq` here, as this seems a general i.MX8MQ thing in the end. Just in the comment it should be a bit more clear, where this MaaxBoard belongs to. In the context of this PR it's obvious, but if somebody with a different context reads this comment one day, it might look odd.

---

Thanks, fixed.